src/main/scala/com/feedzai/cosytest/wrapper/DockerComposeJavaSetup.scala

```diff
@@ -63,31 +64,110 @@ case class DockerComposeJavaSetup(
     *
     * @return true if setup network and containers were removed with success otherwise false
     */
-  def cleanUp: Boolean = dockerSetup.cleanUp()
+  def cleanup: Boolean = dockerSetup.cleanup()
 
+  /**
+    * Returns the port open in localhost mapped to container port.
+    *
+    * @param containerId
+    * @param port exposed from container
+    * @return string of open localhost port. Empty if a failure occurs
+    */
   def getContainerMappedPort(containerId: String, port: Int): String =
     dockerSetup.getContainerMappedPort(containerId, port)
 
+  /**
+    * Returns the list of container Ids associated to service.
+    *
+    * @param serviceName string
+    * @return list of container ids. If a failure occurs an empty list is returned
+    */
   def getServiceContainerIds(serviceName: String) =
     new util.ArrayList[String](dockerSetup.getServiceContainerIds(serviceName).asJava)
 
+  /**
+    * Returns the list of container IP Addresses associated to service.
+    *
+    * @param serviceName string
+    * @return list of container ips.
+    */
+  def getServiceContainerAddresses(serviceName: String) =
+    new util.ArrayList[InetAddress](dockerSetup.getServiceContainerAddresses(serviceName).asJava)
 
+  /**
+    * Returns the list of container Ids associated to the project.
+    *
+    * @return list of container ids. If a failure occurs an empty list is returned.
+    */
   def getProjectContainerIds = new util.ArrayList[String](dockerSetup.getProjectContainerIds().asJava)
 
+  /**
+    * Returns the list of container IPs associated to the project.
+    *
+    * @return list of container ips.
+    */
+  def getProjectContainerIps() = new util.ArrayList[InetAddress](dockerSetup.getProjectContainerIps().asJava)
```

Comment:
I remember commenting on this regarding changing to `getProjectContainerAddressess`...
Also, in some places we call this a project and in some a setup.